specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.1/keys.json

```diff
@@ -0,0 +1,1731 @@
+              },
+              {
+                "value": "RS512",
+                "description": "RSASSA-PKCS1-v1_5 using SHA-512, as described in https://tools.ietf.org/html/rfc7518"
+              },
+              {
+                "value": "RSNULL",
+                "description": "Reserved"
+              },
+              {
+                "value": "ES256",
+                "description": "ECDSA using P-256 and SHA-256, as described in https://tools.ietf.org/html/rfc7518."
+              },
+              {
+                "value": "ES384",
+                "description": "ECDSA using P-384 and SHA-384, as described in https://tools.ietf.org/html/rfc7518"
+              },
+              {
+                "value": "ES512",
+                "description": "ECDSA using P-521 and SHA-512, as described in https://tools.ietf.org/html/rfc7518"
+              },
+              {
+                "value": "ES256K",
+                "description": "ECDSA using P-256K and SHA-256, as described in https://tools.ietf.org/html/rfc7518"
+              }
+            ]
+          }
+        },
+        "value": {
+          "type": "string",
+          "format": "base64url"
+        }
+      },
+      "description": "The key operations parameters.",
+      "required": [
+        "alg",
+        "value"
+      ]
+    },
+    "KeyVerifyParameters": {
+      "properties": {
+        "alg": {
+          "x-ms-client-name": "algorithm",
+          "type": "string",
+          "minLength": 1,
+          "description": "The signing/verification algorithm. For more information on possible algorithm types, see JsonWebKeySignatureAlgorithm.",
+          "enum": [
+            "PS256",
+            "PS384",
+            "PS512",
+            "RS256",
+            "RS384",
+            "RS512",
+            "RSNULL",
+            "ES256",
+            "ES384",
+            "ES512",
+            "ES256K"
+          ],
+          "x-ms-enum": {
+            "name": "JsonWebKeySignatureAlgorithm",
+            "modelAsString": true
+          }
+        },
+        "digest": {
+          "type": "string",
+          "format": "base64url",
+          "description": "The digest used for signing."
+        },
+        "value": {
+          "x-ms-client-name": "signature",
+          "type": "string",
+          "format": "base64url",
+          "description": "The signature to be verified."
+        }
+      },
+      "description": "The key verify parameters.",
+      "required": [
+        "alg",
+        "digest",
+        "value"
+      ]
+    },
+    "KeyUpdateParameters": {
+      "properties": {
+        "key_ops": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "enum": [
+              "encrypt",
+              "decrypt",
+              "sign",
+              "verify",
+              "wrapKey",
+              "unwrapKey",
+              "import"
```

Comment:
Any key with the key op 'import' is a special type of key, a KEK (Key Encryption Key). It can only be created in Key Vault from scratch. No one should be able to update a key with an 'import' key op if it's not created from scratch.

Please remove 'import' from KeyUpdateParameters.

Reply:
Agree. 'import' can only be present (as only value) when creating a new key. You cannot change an existing key and add 'import' to its key_ops.